pkg/objectupdate/rte/rte.go

```diff
@@ -146,11 +147,12 @@ func DaemonSetArgs(ds *appsv1.DaemonSet, conf nropv1.NodeGroupConfig) error {
 		flags.SetOption("--sleep-interval", refreshPeriod)
 	}
 
-	pfpEnabled := isPodFingerprintEnabled(&conf)
+	pfpEnabled, pfpRestricted := isPodFingerprintEnabled(&conf)
 	klog.V(2).InfoS("DaemonSet update: pod fingerprinting status", "daemonset", ds.Name, "enabled", pfpEnabled)
 	if pfpEnabled {
 		flags.SetToggle("--pods-fingerprint")
 		flags.SetOption("--pods-fingerprint-status-file", filepath.Join(pfpStatusDir, "dump.json"))
+		flags.SetOption("--pods-fingerprint-unrestricted", strconv.FormatBool(!pfpRestricted)) // note the "not"!
```

Comment:
Perhaps it is better to have a variable name that is more expressive here.  It is not very descriptive and it is not very evident from here what is the restriction about?  How about `--pods-fingerprint-exclusive-resource-only` ?

Reply:
https://github.com/k8stopologyawareschedwg/resource-topology-exporter/issues/177